extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/OtlpExporterConfig.java

```diff
@@ -27,6 +27,17 @@
         @ConfigItem()
         public Optional<String> endpoint;
 
+        /**
+         * Key-value pairs to be used as headers associated with gRPC requests.
+         * The format is the same as for the {@code OTEL_EXPORTER_OTLP_HEADERS} environment variable,
+         * a coma-separated list of key-value pairs separated by the "=" character.
+         * See <a href=
+         * "https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specifying-headers-via-environment-variables">
+         * Specifying headers</a> for more details.
+         */
+        @ConfigItem()
```

Comment:
It would be good if we can align how we handle lists for OpenTelemetry.

Can you modify it to use a config item as done here: https://github.com/quarkusio/quarkus/blob/main/extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/TracerRuntimeConfig.java#L20

Reply:
Please see the updated code based on `Optional<List<String>> headers`